### PR TITLE
2020-04-22-Noisemaker - stansh

### PR DESCRIPTION
**Pull request by stansh**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+   <button type='button' onclick="play(7)">Play C </button>
+
+ </body>
+
+   <script>
+      var audioCtx = new (window.AudioContext || window.webkitAudioContext);
+
+      // musical notes frequency list
+      var freq = [261.63, 293.66, 329.63, 349.23, 392.00, 440.00, 493.88, 523.25]
+      var sound
```
Comment (zeinabfarag-zz) [inline index.html:24]: variable not used

File: index.html [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+      
+      function play(i) {
+         var sine = audioCtx.createOscillator();
```
Comment (zeinabfarag-zz) [inline index.html:27]: i would try to refrain from using var and switch to either let or const

**Review by zeinabfarag-zz — COMMENTED** [review]
It would be cool if there was a deployed version to see how it works but overall good job!

File: index.html (hunk 1 above)
Comment (stansh) [reply index.html:24]: Sorry first time doing this:
https://htmlpreview.github.io/?https://github.com/MintbeanHackathons/2020-04-22-Noisemaker/blob/1ee322e0405557fdadb293ed1fdc6183cf6ed1e3/index.html

Comment (stansh) [reply index.html:24]: I was going to add 3 more radio buttons for sound types but ran out of time

File: index.html (hunk 2 above)
Comment (stansh) [reply index.html:27]: Got it. Thanks

**Pull request closed without merging**